prometheus/alertmanager.yml

```diff
@@ -0,0 +1,13 @@
+#global:
```

Comment:
Unfortunately, prometheus splits its configuration across many files... its kind of annoying.  This file is exclusively for alert managers, that is places to send alerts.  I've configured slack here, but we can actually configure SMTP as well (for instance, emailing harry when certain things go wrong).